core/src/main/scala/org/apache/spark/sql/CachedDataFrame.scala

```diff
@@ -412,7 +412,6 @@ object CachedDataFrame
     val buffer = new Array[Byte](4 << 10) // 4K
     // final output is written to this buffer
     val output = new ByteBufferDataOutput(4 << 10,
-      DirectBufferAllocator.instance(),
```

Comment:
Will cover this change in https://github.com/SnappyDataInc/snappydata/pull/677 as both deal code in similar area.